homeassistant/components/wake_word/models.py

```diff
@@ -0,0 +1,21 @@
+"""Wake word models."""
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class WakeWord:
+    """Wake word model."""
+
+    ww_id: str
+    name: str
+
+
+@dataclass
+class DetectionResult:
+    """Result of wake word detection."""
+
+    ww_id: str
+    """Id of detected wake word"""
+
+    timestamp: int | None
+    """Timestamp of audio chunk with detected wake word"""
```

Comment:
Is this start or end of the wake word ? Or do we just assume that the chunk is just that wake word ?

Reply:
I think this explanation is great and should make it to the docstring here, and also elsewhere where there are comments about keeping audio up until right before the wake word etc.